applications/surgical_scene_recon/operators/image_saver_op.py

```
@@ -0,0 +1,167 @@
+            self.frame_count += 1
+            return
+        
+        # Get the image tensor
+        # Try different possible tensor names
+        tensor = None
+        tensor_name = None
+        for name in ["rendered_rgb", "input", "image", "rgb"]:
+            try:
+                tensor = in_message.get(name)
+                if tensor is not None:
+                    tensor_name = name
```

Ruff (0.14.5)

Local variable `tensor_name` is assigned to but never used

Remove assignment to unused variable `tensor_name`

(F841)